index.html

```diff
@@ -0,0 +1,235 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,300;1,400;1,500;1,600;1,700;1,800&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="main.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header class="header">
+      <ul class="header__links">
+        <li class="header__item header__item--big-gap">
+          <span class="logo">Medical Core</span>
+        </li>
+        <li class="header__item"><a href="#" class="link">Home</a></li>
+        <li class="header__item"><a href="#" class="link">Product</a></li>
+        <li class="header__item"><a href="#" class="link">Pricing</a></li>
+        <li class="header__item header__item--auto-gap">
+          <a href="#" class="link">Contact</a>
+        </li>
+        <li class="header__item header__item--medium-gap">
+          <a class="link link--blue" href="#">Login</a>
+        </li>
+        <li class="header__item">
+          <a href="#" class="button"
+            >JOIN US
+            <img
+              class="button__icon"
+              src="./assets/right-arrow.svg"
+              alt="arrow"
+          /></a>
+        </li>
+      </ul>
+    </header>
+    <main>
+      <section class="banner">
+        <div class="banner__card">
+          <span class="tag">Welcome</span>
+          <h1 class="title title--lg banner__title">
+            A Great Place to Receive Care
+          </h1>
+          <p class="text text--lg">Get your best looking smile now!</p>
+          <div class="banner__button-container">
+            <a href="#" class="button">Join Us</a>
+            <a href="#" class="button button--secondary">Learn More</a>
+          </div>
+        </div>
+      </section>
+      <section class="section experts">
```

Comment:
Use one block for this class or use section-experts

Reply:
I was trying to use to block on the same element.